### PR TITLE
Removes Bombgers, by vibe checking your TTV bomb in the deep frying process

### LINKED ISSUE #71547: TTVs can be deepfried and hidden in food
## Testmerges:
[#71474: 'Revert "Unit tests radio, saymode, and language prefix keys"' by MrStonedOne at commit 19d0ba1b7e](https://www.github.com/tgstation/tgstation/pull/71474)
[#71527: 'Partially reverts moving up/down whilst in a movable from #70431' by Watermelon914 at commit bb1ccc9005](https://www.github.com/tgstation/tgstation/pull/71527)

## Reproduction:

1. Make a TTV bomb
2. Deep fry it
3. Put the fried bomb in a custom food
4. Small, concealable, renamable bomb hidden in a burger

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

TTVs now open their valve when deep fried. This either KERBLOOEYS the kitchen, or proves the TTV is inert, and as such we don't care if it goes in custom food or whatever. But that's a separate problem, letting larger items enter smaller foods, or rather letting smaller foods stay small when large items enter them

Fixes #71547
Closes #71548

## Why It's Good For The Game

God never intended for TTVs to be turned into easily concealed, explosively delicious food. But I like my solution more than a blacklist~

## Changelog

:cl:
fix: TTVs can have an explosive reaction from deepfrying.
/:cl:

### CODE REVIEW

File: code/game/objects/items/food/deepfried.dm [hunk 1]
```diff
@@ -36,6 +36,11 @@
 	if(!(SEND_SIGNAL(fried, COMSIG_ITEM_FRIED, src) & COMSIG_FRYING_HANDLED)) //If frying is handled by signal don't do the defaault behavior.
 		fried.forceMove(src)
 
+	if(istype(fried, /obj/item/transfer_valve))
+		var/obj/item/transfer_valve/valve = fried
+		log_bomber(null, "TTV valve opened via deepfrying", src, "last fingerprints = [valve.fingerprintslast]")
+		valve.toggle_valve()
+		///if it doesn't blow up, congratulations! it's inert anyways, so we don't care if it's deep fried anymore
```
Comment (Mothblocks) [inline code/game/objects/items/food/deepfried.dm:43]:
```suggestion
		//if it doesn't blow up, congratulations! it's inert anyways, so we don't care if it's deep fried anymore
```

Not a doc comment

File: code/game/objects/items/food/deepfried.dm [hunk 2]
```diff
@@ -36,6 +36,11 @@
 	if(!(SEND_SIGNAL(fried, COMSIG_ITEM_FRIED, src) & COMSIG_FRYING_HANDLED)) //If frying is handled by signal don't do the defaault behavior.
```
Comment (Mothblocks) [inline code/game/objects/items/food/deepfried.dm:36]: Wait, shouldn't we use the signal instead?

**Comment by Mothblocks** [comment]
Ok lets merge #71551 first I guess

File: code/game/objects/items/food/deepfried.dm (hunk 1 above)
Comment (tralezab) [reply code/game/objects/items/food/deepfried.dm:43]: it is important to note exactly WHY it's important but yes, I'll change this

File: code/game/objects/items/food/deepfried.dm (hunk 2 above)
Comment (tralezab) [reply code/game/objects/items/food/deepfried.dm:36]: fdf3b9e737eb2dca09fbfbb2

**Comment by Mothblocks** [comment]
Actually if #71551 preserves the signal then we shouldn't be blocked on it

**Comment by Mothblocks** [comment]
Oh no it removes it lol

**Comment by Mothblocks** [comment]
He brought back the signal so you should be fine to use it

**Comment by tralezab** [comment]
cool beans

**Pull request merged**